### ISSUE TITLE
False positives for CKV_AZURE_65 and CKV_AZURE_66

### ISSUE THREAD

**Issue by askpatrickw** [bridgecrewio/checkov#5110]
**Describe the issue**
CKV_AZURE_65 and CKV_AZURE_66 are reporting failed even though the settings are correct.

**Examples**

```terraform

resource "azurerm_linux_web_app" "app-wap-cms" {
  name                = "${local.environment}-wap-${local.region}-${local.application}-${local.incremental}"
  location            = azurerm_resource_group.app-rsg-cms.location
  resource_group_name = azurerm_resource_group.app-rsg-cms.name
  service_plan_id     = azurerm_service_plan.app-asp-cms.id
  identity {
    type = "SystemAssigned"
  }
  logs {
    detailed_error_messages_enabled = true
    failed_request_tracing_enabled  = true
    http_logs {
      retention_in_days = 4
      retention_in_mb   = 10
    }
  }
```

**Version (please complete the following information):**

2.3.247

**Additional context**

I see in the rules, linked below `'logs/[0]/failed_request_tracing_enabled'` should it be `'logs/failed_request_tracing_enabled'` ?

https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/azure/AppServiceDetailedErrorMessagesEnabled.py#L14

and

https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/azure/AppServiceEnableFailedRequest.py#L13

**Comment by askpatrickw** [comment]
Thanks for you assistance. 
We misinterpreted the code in the check.
